src/types/json.ts

```diff
@@ -13,6 +15,9 @@ export const fetchProcessing = (data: any) => JSON.parse(data);
 
 export const validate = TypeUtils.validate.checkRequired((value) => {
 	try {
+		// Disallow primitives
+		assert(typeof value === 'object');
```

Comment:
`assert` is assuming a nodejs module whereas sbvr-types is technically browser compatible and should stay that way. Aside from that the errors from this are returned to the use in the form of (from memory) `${fieldName}: ${errMessage}`, eg for this it would end up as `contract: cannot be turned into JSON: {"foo":"bar"}` which would be confusing to say the least so I would prefer an explicit error like ``throw new Error(`is not an object/array: ${typeof value}`)`` that makes it much easier for users of the API to know/fix the issue

Reply:
Done